### DISCUSSION TITLE
Stats API made public?

### DISCUSSION THREAD

**Discussion by M0TZO** [magicbug/Cloudlog#1697] [General]
Hi,

Could the JSON feed at /index.php/api/statistics be made public (even with a slug?) so can use with my Inkyphat screen again?

I like seeing my stats from there.

Thanks in advance (_not urgent_)

**Comment by M0TZO** [comment]
Could this be looked into?

I know I'm probably one of the few using this feature, but was useful.

Or is there already a way to make it work without a login?

**Comment by phl0** [comment]
There is a PR for that: https://github.com/magicbug/Cloudlog/pull/2090. Please test, report and close here if successful. Thank you :)

**Comment by M0TZO** [comment]
Thanks for this. Some tinkering and got the DEV branch working and I've put a pic below of my E-Ink (InkyPhat) screen.

Hopefully will be in main branch in the future, but I'm happy for now.

Do wonder if anyone else has found and used this?

![image](https://user-images.githubusercontent.com/2846980/235116658-9bbdd38c-9165-4cb9-8b9b-91018d1ee931.png)

**Comment by AndreasK79** [comment]
Closing as this is integrated into dev.
